website/content/docs/app-config/dynamic.mdx

```diff
@@ -94,8 +94,7 @@ Configuration sources sometimes support global settings for their behavior or
 require global settings to be set for them to function at all.
 For example, [Vault](/waypoint/plugins/vault#vault-configsourcer) has some global
 settings around how to authenticate. Without setting these, Vault configurations
-in applications will not work. These _source settings_ are set at the server
```

Comment:
Did you intend to delete this? It still seems relevant.

Reply:
I misread this section as a note about _all_ config source settings, not just global - you are right, this does still pertain to globally-scoped sources! Will add it back.